packages/babel-parser/test/fixtures/es2015/yield/parameter-name-arrow-inside-generator-1/output.json

```diff
@@ -2,7 +2,7 @@
   "type": "File",
   "start":0,"end":35,"loc":{"start":{"line":1,"column":0},"end":{"line":3,"column":1}},
   "errors": [
-    "SyntaxError: Can not use 'yield' as identifier inside a generator (2:3)",
+    "SyntaxError: yield is not allowed in generator parameters (2:3)",
```

Comment:
https://github.com/babel/babel/blob/d51aa6d76177b544590cdfe3868f9f4d33d8813d/packages/babel-parser/test/fixtures/es2015/yield/parameter-default-inside-arrow-inside-generator-1/input.js#L1-L3

The error message seems a bit misleading as it is not a generator, although it is indeed in function parameters. V8 has changed this error message to "Yield expression not allowed in formal parameter" ([code](https://source.chromium.org/chromium/chromium/src/+/master:v8/src/common/message-template.h;l=543;drc=bd16dc8f9d636755d54ee27bc95435afd258552d), I suggest we align with V8 and update the error in following PRs.